.github/workflows/release-binaries.yml

```diff
@@ -18,29 +18,31 @@ jobs:
   # Each time this workflow is executed, a build will be triggered to create a new image
   # with the corresponding tags using information from git
 
-  # The image will be named `zebra:<semver>`
-  build:
-    name: Build Release Docker
+  # The image will be named `zebra:<semver>.experimental`
+  build-experimental:
```

Comment:
These are swapped to make sure tags with the same name point to the production image.